airbyte-integrations/connectors/source-mysql-v2/src/main/kotlin/io/airbyte/integrations/source/mysql/MysqlSourceConfigurationJsonObject.kt

```diff
@@ -52,7 +52,7 @@ import jakarta.inject.Singleton
 @Singleton
 @ConfigurationProperties(CONNECTOR_CONFIG_PREFIX)
 @SuppressFBWarnings(value = ["NP_NONNULL_RETURN_VIOLATION"], justification = "Micronaut DI")
-class MysqlSourceConfigurationJsonObject : ConfigurationJsonObjectBase() {
+class MysqlSourceConfigurationJsonObject : ConfigurationSpecification() {
```

Comment:
The DB sources team can take care of renaming this class and any others in source-mysql-v2 but could you check again that you didn't forget to rename anything in the Bulk CDK? Please do a full project string search for `ConfigurationJsonObject`. IntelliJ is smart, but not that smart.

Reply:
```
jschmidt@Johnny-Schmidt-MacBook-Pro---C2WMR2G02N airbyte % git grep 'ConfigurationJsonObject' | grep -v airbyte-integrations/connectors
jschmidt@Johnny-Schmidt-MacBook-Pro---C2WMR2G02N airbyte %
```